tools/bump_builder.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
I think we could simplify this logic and erhaps even remove this file if we had the SHA in a single location, defined in a variable and being used everywhere.

as a reference, this is upstream: https://github.com/istio/common-files/blob/master/files/common/scripts/setup_env.sh#L62